pkgs/by-name/co/colorized-logs/package.nix

```diff
@@ -0,0 +1,35 @@
+{ cmake
+, fetchFromGitHub
+, lib
+, stdenv
+, ...
+}:
+
+let
+  pname = "colorized-logs";
+  owner = "kilobyte";
+  version = "2.6";
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-QiZeIYeIWA3C7wYi2G2EItdW+jLjVrCbIYllur/RtY8=";
+  };
+
+  meta = {
+    description = "Tools for logs with ANSI color";
+    homepage = "https://github.com/${owner}/${pname}";
+    changelog = "https://github.com/${owner}/${pname}/blob/v${version}/ChangeLog";
+    license = with lib.licenses; [ mit ];
+    maintainers = with lib.maintainers; [ CobaltCause ];
+  };
```

Comment:
Add platforms and mainProgram

Reply:
> Add platforms

Okay.

> and mainProgram

This package has 4-5 different binaries depending on whether you count symlinks, I'm not sure that's sensible here.

Reply:
I believe `pipetty` is a nice choice.

Reply:
I thought it would be best to make users consciously decide which program they want to run, rather than having `nix run` (or similar) dictate a "random" default